### PR TITLE
Web4542 update coding styles

### PR DESCRIPTION
**Pull request by lukefrake**
#### What are the links to relevant tickets?
WEB-4542
#### What does this PR do?
Add make-up
Make pass hinting linting etc
#### What unit tests does this PR have?
N/A
#### How should a developer review this?
Pull down & run tests / see travis run
#### Any background context you want to provide?
Part of something bigger
#### Screenshots (if appropriate)
---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [x] I've tested this cross browser for any visual changes.
- [x] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.

#### Reviewer 1 @kevinhodges 
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running successfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

#### Reviewer 2
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve witnessed the tests running succesfully.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

**Comment by lukefrake** [comment]
The tests fail on this repo because it tries to set up a local Elephant database.

I've made a jira to fix the problem, but I won't be fixing it as part of this.

**Comment by kevinhodges** [comment]
Can we strip Phil's details from the main lib file pls

File: package.json [hunk 1]
```diff
@@ -27,10 +27,13 @@
   "devDependencies": {
     "chai": "1.9.2",
     "chai-as-promised": "4.1.1",
-    "grunt": "0.4.3",
-    "grunt-contrib-jshint": "0.10.0",
-    "grunt-jscs": "0.6.2",
-    "grunt-mocha-test": "0.10.2",
+    "grunt": "0.4.5",
+    "mocha": "1.20.1",
+    "make-up": "^1.0.3",
```
Comment (philmeehan) [inline package.json]: hat

**Comment by lukefrake** [comment]
Removed Phils & Your details, when this branch is merged I'll burn the history.

File: test/pluginElephantTest.js [hunk 2]
```diff
@@ -57,7 +46,7 @@ describe( 'pluginElephant', function() {
 	describe( '.makeItSo()', function() {
 
 		it( 'should expose makeItSo as a function', function( done ) {
-			server.plugins['plugin-elephant'].makeItSo.should.be.a.function;
+			server.plugins['plugin-elephant'].makeItSo.should.be.a.function; // jshint ignore:line
```
Comment (philmeehan) [inline test/pluginElephantTest.js]: Can we remove these ignores here and at the top of the file have these two lines

/* jslint node: true */
/* jshint -W030 */

(keep consistency with other repos and will just ignore the one rule not the rest)

File: test/pluginElephantTest.js [hunk 3]
```diff
@@ -49,7 +38,7 @@ describe( 'pluginElephant', function() {
 
 	describe( '.register()', function() {
 		it( 'should allow us to access the plugin off the hapi server', function( done ) {
-			server.plugins['plugin-elephant'].should.not.be.undefined;
+			server.plugins['plugin-elephant'].should.not.be.undefined; // jshint ignore:line
```
Comment (philmeehan) [inline test/pluginElephantTest.js]: same as comment below

**Comment by lukefrake** [comment]
Thanks @philmeehan  - I didn't know that you could do that, updated now.

File: test/pluginElephantTest.js [hunk 4]
```diff
@@ -1,21 +1,11 @@
...
-* @description Root test file, there's a Grunt task to execute all this shizzle, powered by Mocha
-* @since Tue Jul 01 2014
-* @author Phil Meehan <philip.meehan@holidayextras.com>
-*/
-
 var Hapi = require( 'hapi' );
 var chai = require( 'chai' );
 var chaiAsPromised = require( 'chai-as-promised' );
 chai.use( chaiAsPromised );
-var should = chai.should();
```
Comment (philmeehan) [inline test/pluginElephantTest.js]: Add these lines back - should fix linting and running the tests locally.
```
var should;
should = chai.should();
```

**Pull request merged**